tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -112,13 +132,116 @@ public static Security Instance
 
         internal SecuritySettings Settings => _settings;
 
-        internal string DdlibWafVersion => _waf?.Version;
+        internal string? DdlibWafVersion => _waf?.Version;
+
+        internal void SubscribeToChanges(params string[] productNames)
+        {
+            void RcmSubscription(RemoteConfigurationManager remoteConfigurationManager)
+            {
+                _remoteConfigurationManager ??= remoteConfigurationManager;
+                if (_rcmSubscription is null)
+                {
+                    _rcmSubscription = remoteConfigurationManager.SubscribeToChanges(
+                        UpdateFromRcm,
+                        productNames);
+                }
+                else
+                {
+                    _rcmSubscription.SubscribeProducts(productNames);
+                }
+            }
+
+            if (_remoteConfigurationManager is null)
```

Comment:
Not in the scope of this PR but we need to resolve the fact that remoteConfigManager should never be null